Ofqual.Recognition.Citizen.API.Infrastructure/Repositories/ApplicationRepository.cs

```diff
@@ -0,0 +1,32 @@
+using System.Data;
+using Dapper;
+using Ofqual.Recognition.Citizen.API.Core.Models;
+
+namespace Ofqual.Recognition.Citizen.API.Infrastructure.Repositories;
+
+public class ApplicationRepository : IApplicationRepository
+{
+    private readonly IDbTransaction _dbTransaction;
+
+    public ApplicationRepository(IDbTransaction dbTransaction)
+    {
+        _dbTransaction = dbTransaction;
+    }
+
+    public async Task<Application> CreateApplication()
```

Comment:
This comment applies to all of your repositories - you must wrap your database calls in try-catches and handle these gracefully. I'm aware that there's catch-alls on the controllers, but it's always better to catch earlier if you can just so you can log out _where_ the function bombed out. It's totally ok to log out and then rethrow in the catch too if it's a fatal error, but even a log to say something like "CreateApplication in ApplicationRepository" failed is _really_ useful when debugging in production. If you're gonna add Serilog, make sure you use it to it's fullest! 😝